<!DOCTYPE html>
<html><head><title>Exception</title><link rel="stylesheet" href="../../styles.css"/><script src="../../scripts.js"></script></head><body onload="ro();">
<div class="rH">68 references to Exception</div><div class="rA">Microsoft.PowerShell.Commands.Utility (2)</div><div class="rG" id="Microsoft.PowerShell.Commands.Utility"><div class="rF"><div class="rN">commands\utility\ShowCommand\ShowCommand.cs (1)</div>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/ShowCommand/ShowCommand.cs.html#246"><b>246</b>errorString.Append(error.<i>Exception</i>.Message);</a>
</div>
<div class="rF"><div class="rN">commands\utility\WebCmdlet\Common\InvokeRestMethodCommand.Common.cs (1)</div>
<a href="../../Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/InvokeRestMethodCommand.Common.cs.html#201"><b>201</b>exRef = error.<i>Exception</i>;</a>
</div>
</div>
<div class="rA">System.Management.Automation (66)</div><div class="rG" id="System.Management.Automation"><div class="rF"><div class="rN">DscSupport\CimDSCParser.cs (1)</div>
<a href="../DscSupport/CimDSCParser.cs.html#60"><b>60</b>innerException = powerShell.Streams.Error[0].<i>Exception</i>;</a>
</div>
<div class="rF"><div class="rN">engine\cmdlet.cs (2)</div>
<a href="../engine/cmdlet.cs.html#1729"><b>1729</b>else if (errorRecord.<i>Exception</i> != null)</a>
<a href="../engine/cmdlet.cs.html#1731"><b>1731</b>throw errorRecord.<i>Exception</i>;</a>
</div>
<div class="rF"><div class="rN">engine\DefaultCommandRuntime.cs (4)</div>
<a href="../engine/DefaultCommandRuntime.cs.html#50"><b>50</b>if (errorRecord.<i>Exception</i> != null)</a>
<a href="../engine/DefaultCommandRuntime.cs.html#51"><b>51</b>throw errorRecord.<i>Exception</i>;</a>
<a href="../engine/DefaultCommandRuntime.cs.html#236"><b>236</b>if (errorRecord.<i>Exception</i> != null)</a>
<a href="../engine/DefaultCommandRuntime.cs.html#238"><b>238</b>throw errorRecord.<i>Exception</i>;</a>
</div>
<div class="rF"><div class="rN">engine\ErrorPackage.cs (12)</div>
<a href="../engine/ErrorPackage.cs.html#269"><b>269</b>if (_errorRecord.<i>Exception</i> != null)</a>
<a href="../engine/ErrorPackage.cs.html#272"><b>272</b>return _errorRecord.<i>Exception</i>.GetType().Name;</a>
<a href="../engine/ErrorPackage.cs.html#738"><b>738</b>/// &lt;see cref=&quot;System.Management.Automation.ErrorRecord.<i>Exception</i>&quot;/&gt;</a>
<a href="../engine/ErrorPackage.cs.html#749"><b>749</b>/// &lt;see cref=&quot;System.Management.Automation.ErrorRecord.<i>Exception</i>&quot;/&gt;.</a>
<a href="../engine/ErrorPackage.cs.html#1210"><b>1210</b>RemotingEncoder.AddNoteProperty&lt;Exception&gt;(dest, &quot;Exception&quot;, () =&gt; <i>Exception</i>);</a>
<a href="../engine/ErrorPackage.cs.html#1404"><b>1404</b>&amp;&amp; (errorRecord.<i>Exception</i> is ParentContainsErrorRecordException))</a>
<a href="../engine/ErrorPackage.cs.html#1410"><b>1410</b>_error = errorRecord.<i>Exception</i>;</a>
<a href="../engine/ErrorPackage.cs.html#1694"><b>1694</b>if (<i>Exception</i> != null)</a>
<a href="../engine/ErrorPackage.cs.html#1696"><b>1696</b>if (!string.IsNullOrEmpty(<i>Exception</i>.Message))</a>
<a href="../engine/ErrorPackage.cs.html#1698"><b>1698</b>return <i>Exception</i>.Message;</a>
<a href="../engine/ErrorPackage.cs.html#1701"><b>1701</b>return <i>Exception</i>.ToString();</a>
<a href="../engine/ErrorPackage.cs.html#1779"><b>1779</b>/// &lt;see cref=&quot;System.Management.Automation.ErrorRecord.<i>Exception</i>&quot;/&gt;</a>
</div>
<div class="rF"><div class="rN">engine\ExecutionContext.cs (2)</div>
<a href="../engine/ExecutionContext.cs.html#867"><b>867</b>if (er1 != null &amp;&amp; objAsErrorRecord != null &amp;&amp; er1.<i>Exception</i> == objAsErrorRecord.<i>Exception</i>)</a>
</div>
<div class="rF"><div class="rN">engine\InitialSessionState.cs (1)</div>
<a href="../engine/InitialSessionState.cs.html#3228"><b>3228</b>firstError = er.<i>Exception</i>;</a>
</div>
<div class="rF"><div class="rN">engine\Modules\ModuleCmdletBase.cs (4)</div>
<a href="../engine/Modules/ModuleCmdletBase.cs.html#3069"><b>3069</b>new PSInvalidOperationException(errorRecord.<i>Exception</i>.Message, errorRecord.<i>Exception</i>,</a>
<a href="../engine/Modules/ModuleCmdletBase.cs.html#3815"><b>3815</b>if (error != null &amp;&amp; error.<i>Exception</i> != null)</a>
<a href="../engine/Modules/ModuleCmdletBase.cs.html#3817"><b>3817</b>mm = new MissingMemberException(error.<i>Exception</i>.Message);</a>
</div>
<div class="rF"><div class="rN">engine\Modules\RemoteDiscoveryHelper.cs (3)</div>
<a href="../engine/Modules/RemoteDiscoveryHelper.cs.html#378"><b>378</b>else if (innerErrorRecord.<i>Exception</i> != null &amp;&amp; innerErrorRecord.<i>Exception</i>.Message != null)</a>
<a href="../engine/Modules/RemoteDiscoveryHelper.cs.html#380"><b>380</b>innerErrorMessage = innerErrorRecord.<i>Exception</i>.Message;</a>
</div>
<div class="rF"><div class="rN">engine\MshCommandRuntime.cs (3)</div>
<a href="../engine/MshCommandRuntime.cs.html#2081"><b>2081</b>if (errorRecord.<i>Exception</i> != null</a>
<a href="../engine/MshCommandRuntime.cs.html#2082"><b>2082</b>&amp;&amp; string.IsNullOrEmpty(errorRecord.<i>Exception</i>.StackTrace))</a>
<a href="../engine/MshCommandRuntime.cs.html#2086"><b>2086</b>throw errorRecord.<i>Exception</i>;</a>
</div>
<div class="rF"><div class="rN">engine\remoting\client\Job.cs (1)</div>
<a href="../engine/remoting/client/Job.cs.html#758"><b>758</b>if (!(errorRecord.<i>Exception</i> is RuntimeException runtimeException))</a>
</div>
<div class="rF"><div class="rN">engine\remoting\client\JobManager.cs (1)</div>
<a href="../engine/remoting/client/JobManager.cs.html#366"><b>366</b>ex = powerShell.ErrorBuffer[0].<i>Exception</i>;</a>
</div>
<div class="rF"><div class="rN">engine\remoting\client\remoterunspace.cs (9)</div>
<a href="../engine/remoting/client/remoterunspace.cs.html#1942"><b>1942</b>re.ErrorRecord.<i>Exception</i>?.Message,</a>
<a href="../engine/remoting/client/remoterunspace.cs.html#1952"><b>1952</b>(re.ErrorRecord.<i>Exception</i> != null) ? re.ErrorRecord.<i>Exception</i>.Message : string.Empty);</a>
<a href="../engine/remoting/client/remoterunspace.cs.html#2274"><b>2274</b>Exception e = ps.ErrorBuffer[0].<i>Exception</i>;</a>
<a href="../engine/remoting/client/remoterunspace.cs.html#2886"><b>2886</b>Exception e = ps.ErrorBuffer[0].<i>Exception</i>;</a>
<a href="../engine/remoting/client/remoterunspace.cs.html#3004"><b>3004</b>throw new PSNotSupportedException(RunspaceStrings.NotSupportedOnRestrictedRunspace, error.<i>Exception</i>);</a>
<a href="../engine/remoting/client/remoterunspace.cs.html#3066"><b>3066</b>throw new PSNotSupportedException(RunspaceStrings.NotSupportedOnRestrictedRunspace, error.<i>Exception</i>);</a>
<a href="../engine/remoting/client/remoterunspace.cs.html#3070"><b>3070</b>throw new PSInvalidOperationException(error.<i>Exception</i>.Message, error.<i>Exception</i>);</a>
</div>
<div class="rF"><div class="rN">engine\remoting\commands\CustomShellCommands.cs (1)</div>
<a href="../engine/remoting/commands/CustomShellCommands.cs.html#5267"><b>5267</b>ThrowTerminatingError(new ErrorRecord(e.ErrorRecord.<i>Exception</i>, &quot;CouldNotFindSessionConfiguration&quot;, ErrorCategory.ObjectNotFound, this.ConfigurationName));</a>
</div>
<div class="rF"><div class="rN">engine\remoting\commands\InvokeCommandCommand.cs (4)</div>
<a href="../engine/remoting/commands/InvokeCommandCommand.cs.html#1990"><b>1990</b>(errorRecord.<i>Exception</i> != null) &amp;&amp;</a>
<a href="../engine/remoting/commands/InvokeCommandCommand.cs.html#1991"><b>1991</b>(errorRecord.<i>Exception</i>.InnerException != null))</a>
<a href="../engine/remoting/commands/InvokeCommandCommand.cs.html#1993"><b>1993</b>PSDirectException ex = errorRecord.<i>Exception</i>.InnerException as PSDirectException;</a>
<a href="../engine/remoting/commands/InvokeCommandCommand.cs.html#1996"><b>1996</b>streamObject.Value = new ErrorRecord(errorRecord.<i>Exception</i>.InnerException,</a>
</div>
<div class="rF"><div class="rN">engine\remoting\commands\newrunspacecommand.cs (4)</div>
<a href="../engine/remoting/commands/newrunspacecommand.cs.html#552"><b>552</b>if ((errorRecord.<i>Exception</i> != null) &amp;&amp;</a>
<a href="../engine/remoting/commands/newrunspacecommand.cs.html#553"><b>553</b>(errorRecord.<i>Exception</i>.InnerException != null))</a>
<a href="../engine/remoting/commands/newrunspacecommand.cs.html#555"><b>555</b>PSDirectException ex = errorRecord.<i>Exception</i>.InnerException as PSDirectException;</a>
<a href="../engine/remoting/commands/newrunspacecommand.cs.html#558"><b>558</b>errorRecord = new ErrorRecord(errorRecord.<i>Exception</i>.InnerException,</a>
</div>
<div class="rF"><div class="rN">engine\remoting\common\WireDataFormat\EncodeAndDecode.cs (1)</div>
<a href="../engine/remoting/common/WireDataFormat/EncodeAndDecode.cs.html#2069"><b>2069</b>return er.<i>Exception</i>;</a>
</div>
<div class="rF"><div class="rN">engine\remoting\common\WireDataFormat\RemoteHostEncoder.cs (1)</div>
<a href="../engine/remoting/common/WireDataFormat/RemoteHostEncoder.cs.html#289"><b>289</b>return errorRecord.<i>Exception</i>;</a>
</div>
<div class="rF"><div class="rN">engine\runtime\Operations\MiscOps.cs (5)</div>
<a href="../engine/runtime/Operations/MiscOps.cs.html#1200"><b>1200</b>if (rte.ErrorRecord.<i>Exception</i> is System.ArgumentException)</a>
<a href="../engine/runtime/Operations/MiscOps.cs.html#1204"><b>1204</b>rte.ErrorRecord.<i>Exception</i>, File, rte.ErrorRecord.<i>Exception</i>.Message);</a>
<a href="../engine/runtime/Operations/MiscOps.cs.html#1513"><b>1513</b>var exceptionToPass = apse.ErrorRecord.<i>Exception</i>;</a>
<a href="../engine/runtime/Operations/MiscOps.cs.html#1995"><b>1995</b>runtimeException = new RuntimeException(er.ToString(), er.<i>Exception</i>, er) { WasThrownFromThrowStatement = true, WasRethrown = rethrow };</a>
</div>
<div class="rF"><div class="rN">namespaces\LocationGlobber.cs (1)</div>
<a href="../namespaces/LocationGlobber.cs.html#873"><b>873</b>throw errorRecord.<i>Exception</i>;</a>
</div>
<div class="rF"><div class="rN">utils\ExecutionExceptions.cs (1)</div>
<a href="../utils/ExecutionExceptions.cs.html#39"><b>39</b>if (errorRecord.<i>Exception</i> != null)</a>
</div>
<div class="rF"><div class="rN">utils\PowerShellETWTracer.cs (1)</div>
<a href="../utils/PowerShellETWTracer.cs.html#1006"><b>1006</b>Exception exception = errorRecord.<i>Exception</i>;</a>
</div>
<div class="rF"><div class="rN">utils\RuntimeException.cs (4)</div>
<a href="../utils/RuntimeException.cs.html#233"><b>233</b>if (errorRecord.<i>Exception</i> == null)</a>
<a href="../utils/RuntimeException.cs.html#235"><b>235</b>return errorRecord.<i>Exception</i>.Message;</a>
<a href="../utils/RuntimeException.cs.html#259"><b>259</b>return errorRecord.<i>Exception</i>;</a>
<a href="../utils/RuntimeException.cs.html#276"><b>276</b>RuntimeException exception = _errorRecord.<i>Exception</i> as RuntimeException;</a>
</div>
</div>
</body></html>